docs/sources/dashboards/use-dashboards.md

```diff
@@ -19,40 +19,36 @@ keywords:
-
-{{< figure src="/static/img/docs/v50/dashboard_annotated.png" class="no-shadow" width="700px" >}}
-
-- **Zoom out time range** (1)
-- **Time picker dropdown** (2). Access relative time range options, auto refresh options and set custom absolute time ranges.
-- **Refresh option** (3) Click to trigger queries or set auto refresh.
-- **Dashboard panel** (4) Click the panel title to open panel menu.
-- **Graph legend** (5) Change series colors, y-axis and series visibility directly from the legend.
-
-For more details, see [Dashboard header]({{< relref "#dashboard-header" >}}) and [Dashboard rows]({{< relref "#dashboard-rows" >}}).
-
-## Dashboard header
-
-The dashboard header has the following sections.
-
-{{< figure src="/static/img/docs/v50/top_nav_annotated.png" width="450px" >}}
-
-- **Dashboard title** (2): This also opens the dashboard search when clicked.
-- **Add panel** (3): Use this option to add a new panel or row to the current dashboard.
-- **Star dashboard** (4): Use this option to star (or unstar) the current dashboard. Starred dashboards show up on your own home dashboard and in the navigation bar by default. It is a convenient way to mark Dashboards that you're interested in.
-- **Share dashboard** (5): Use this option to share the current dashboard by link or snapshot. You can also export the dashboard definition from the share modal.
-- **Save dashboard** (6): Use this option to save the current dashboard using its current name.
-- **Settings** (7): Use this option to open dashboard settings. Here you change dashboard name, folder, tags as well as manage variables and annotation queries.
-
-## Dashboard panels
-
-The main building block of dashboard is the panel. You add new panels via the `Add panel` button at the top of the dashboard. This will add view at the top where you can choose if you want to add a new panel, row or library panel. Library panels are panels that can be shared (reused) in many dashboards. You can move panels around by just dragging the panel header. To resize panel use the lower right corner.
-
-## Dashboard rows
-
-A dashboard row is a logical divider within a dashboard. It is used to group panels together. Rows can be collapsed or expanded allowing you to hide parts of the dashboard. Panels inside a collapsed row will not issue any queries.
-
-Use the [repeating rows]({{< relref "../panels/add-panels-dynamically/configure-repeating-rows" >}}) to dynamically create rows based on a template variable.
+This topic provides an overview of dashboard features and shortcuts, and describes how to use dashboard search. 
+
+## Dashboard feature overview
+
+The dashboard user interface provides a number of features that you can use to customize the presentation of your data. 
+
+The following image and descriptions highlights all dashboards features.
+
+{{< figure src="/static/img/docs/v91/dashboard-features/dashboard-features.png" width="700px" >}}
+
+- **Grafana home** (1): Click the Grafana home icon to XXX.
```

Comment:
@axelavargas , I'm wondering how to describe what happens when the user clicks the grafana icon.

Reply:
that's a good one 🤔 , technically it will redirect to the home page configured in the Grafana instance

Reply:
Done, thanks!